src/App.jsx

```diff
@@ -24,26 +24,12 @@ export function App() {
 	useEffect(() => {
 		if (listToken) navigate('/list');
 
-		// }
-		/**
-		 * streamListItems` takes a `listToken` so it can communicate
-		 * with our database; then calls a callback function with
-		 * a `snapshot` from the database.
-		 *
-		//  * Refer to `api/firebase.js`.
-		//  */
-		// 	return streamListItems(listToken, (snapshot) => {
-		// 		/**
-		// 		 * Read the documents in the snapshot and do some work
-		// 		 * on them, so we can save them in our React state.
-		// 		 *
-		// 		 * Refer to `api/firebase.js`
-		// 		 */
-		// 		const nextData = getItemData(snapshot);
+		return streamListItems(listToken, (snapshot) => {
+			const nextData = getItemData(snapshot);
```

Comment:
We actually moved the streamListItems(listToken, (snapshot)) to the List component since this is fetching the list items, so we are not sure if our work here is conflicting :D